.github/workflows/ruby-rspec.yml

```diff
@@ -0,0 +1,23 @@
+name: hcloud-ruby ci
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [ '2.7' ]
+
```

Comment:
I would add 3.1 too and set it to continue-on-error
```yaml
strategy:
  matrix:
    ruby-version: ['2.7', '3.1']
continue-on-error: ${{ matrix.ruby == '3.1' }}
```

Reply:
I've also added 3.0.